From deed437699c9749db2dc08ef0a6969239af32545 Mon Sep 17 00:00:00 2001
From: Alexander Graf <agraf@suse.de>
Date: Sat, 19 Jan 2013 11:30:25 +0100
Subject: [PATCH] OMAP: Fix missing cm3xxx.h include
Patch-Mainline: No, too hacky :)

Without this header included, we get the following build failures:

drivers/staging/tidspbridge/core/tiomap_io.c: In function 'sm_interrupt_dsp':
drivers/staging/tidspbridge/core/tiomap_io.c:405:24: error: 'OMAP3430_CM_AUTOIDLE_PLL' undeclared (first use in this function)
drivers/staging/tidspbridge/core/tiomap_io.c:405:24: note: each undeclared identifier is reported only once for each function it appears in
drivers/staging/tidspbridge/core/tiomap_io.c:416:24: error: 'OMAP3430_CM_CLKEN_PLL' undeclared (first use in this function)
drivers/staging/tidspbridge/core/tiomap3430.c: In function 'bridge_brd_start':
drivers/staging/tidspbridge/core/tiomap3430.c:550:24: error: 'OMAP3430_CM_AUTOIDLE_PLL' undeclared (first use in this function)
drivers/staging/tidspbridge/core/tiomap3430.c:550:24: note: each undeclared identifier is reported only once for each function it appears in

Signed-off-by: Alexander Graf <agraf@suse.de>
---
 drivers/staging/tidspbridge/core/tiomap_io.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/staging/tidspbridge/core/tiomap_io.h b/drivers/staging/tidspbridge/core/tiomap_io.h
index a3f19c7..c35b30c 100644
--- a/drivers/staging/tidspbridge/core/tiomap_io.h
+++ b/drivers/staging/tidspbridge/core/tiomap_io.h
@@ -19,6 +19,8 @@
 #ifndef _TIOMAP_IO_
 #define _TIOMAP_IO_
 
+#include "../../../../arch/arm/mach-omap2/cm3xxx.h"
+
 /*
  * Symbol that defines beginning of shared memory.
  * For OMAP (Helen) this is the DSP Virtual base address of SDRAM.
-- 
1.7.12.4

